WORKSPACE.bazel

```diff
@@ -81,8 +81,63 @@ load("@rules_oci//oci:repositories.bzl", "oci_register_toolchains")
 oci_register_toolchains(name = "oci")
 
 load("@rules_oci//oci:pull.bzl", "oci_pull")
+load("//bazel:mainnet-canisters.bzl", "canisters")
 load("//third_party/lmdb:repository.bzl", "lmdb_repository")
-load("mainnet-canisters.bzl", "mainnet_ck_canisters", "mainnet_core_nns_canisters", "mainnet_sns_canisters")
+
+# We use inconsistent filenames and repository references, so we perform a lot
+# of remapping
```

Comment:
Could you say a bit more in this comment: 
1. What exactly are the keys and values?
2. When should one add entries to this map, and what would happen if this maps is not consistent (e.g., due to a typo)?

Reply:
> What exactly are the keys and values?

Let me know if the updated comment makes sense.

> When should one add entries to this map, and what would happen if this maps is not consistent (e.g., due to a typo)?

You'll get a build error early (i.e. during bazel analysis). IMO this is expected so I think not worth adding to the comment. Lmk!